### ISSUE TITLE
Plugin not working at all on Windows. Broke with vitest >= 0.20.0

### ISSUE THREAD

**Issue by falynx** [vitest-dev/vscode#89]
**Describe the bug**
This plugin isn't working at all on Windows with the current version of vitest. Simple test case doesn't run. Spinners everywhere no matter what I do (with and without "watch" mode enabled).

This all appears to have broken after the release of vitest 0.20.0. The test case below works fine with the vitest version used in [the example project](https://github.com/zxch3n/vitest-ext-basic-example), namely 0.12.6. It works all the way up to vitest 0.19.1, and breaks with vitest >= 0.20.0.

Here's the vitest breaking change in [v0.20.0](https://github.com/vitest-dev/vitest/releases/tag/v0.20.0) that probably caused it:

🚨 Breaking Changes
Apply Vite resolving algorithm to node_modules libraries  -  by @sheremet-va in [#1673](https://github.com/vitest-dev/vitest/pull/1673)

**To Reproduce**
1. Create a simple vitest project with: `npm install -D vitest`
2. Create a test file called foo.test.js:
```
import { describe, it, expect } from 'vitest';

describe('suite', () => {
  it('should work', () => {
    expect(4).toBe(4);
  });
});
```
3. Restart VSCode so the vitest plugin recognizes vitest and finds the test.
4. Confirm that vitest actually works on the command line by running: `npx vitest`
5. Click the Test Explorer tab in VSCode and click any of the test buttons.
6. Notice that everything spins. No feedback or output anywhere. Status bar says "Running tests..." forever.

**Expected behavior**
The vitest plugin features should work as they do with [the example project](https://github.com/zxch3n/vitest-ext-basic-example).

**Screenshots**
![image](https://user-images.githubusercontent.com/113862975/193385761-839c09e2-e81d-438e-9f5a-c20262d2befd.png)

**Environment**
- OS: Windows 11
- VSCode version: 1.71.2
- Vitest version: 0.23.4
- Vitest plugin version: 0.2.29

**Comment by anicolao** [comment]
I have a similar (in my opinion most likely identical) issue on Mac OS X, but the behaviour reproduces starting @ `vitest 0.17.0` and up. In the vitest output window, you get an endless stream of `WS Close` messages when it is not working, but when it is working you usually get a `WS Open` after the series of `WS Close` messages and output displays. I surmise that the extension is polling for results and never getting the starting with vitest v17.0

**Comment by beorn** [comment]
Duplicate of #55 ?

**Comment by hiepxanh** [comment]
may be this one ?


> fix: default host to localhost instead of 127.0.0.1 (#8543) (49c0896), closes #8543
